src/lsp/cobol_parser/grammar_tokens_printer.ml

```diff
@@ -77,7 +77,8 @@ let pp_token: token Pretty.printer = fun ppf ->
     | FIXEDLIT (i, sep, d) -> print "FIXED[%s%c%s]" i sep d
     | FLOATLIT (i, sep, d, e) -> print "FLOAT[%s%c%sE%s]" i sep d e
     | INTERVENING_ c -> print "<%c>" c
-    | tok when Text_keywords.is_intrinsic_token tok ->
+    | INTRINSIC_FUNC _ as tok
+    | tok when Text_keywords.is_known_intrinsic_token tok ->
```

Comment:
Notice that guard applies to both cases (`INTRINSIC_FUNC _ as tok` *and* `tok`). In this case you should just duplicate the for for `printing` (actually using `SPECIALIZED_INTRINSIC_FUNC[%s]` in the second case to distinguish the two kinds of tokens).